cpp/daal/src/threading/threading.h

```diff
@@ -25,6 +25,7 @@
 #define __THREADING_H__
 
 #include "services/daal_defines.h"
+#include <cstdint>
```

Comment:
It isn't look like we need this dependency here. In fact you only need to have 'unsigned long *' and 'unsigned long long *' casting in the safe way.

Reply:
removed